### PR TITLE
Fix broken reference in resource estimation

### PR DESCRIPTION
**Pull request by soranjh**
**Summary:**
Fix broken reference in resource estimation.

**Relevant references:**

**Possible Drawbacks:**

**Related GitHub Issues:**

### CODE REVIEW

**Review by KetpuntoG — APPROVED** [review]
Great! Thanks for the fix

File: demonstrations/tutorial_resource_estimation.py [hunk 1]
```diff
@@ -302,8 +302,7 @@
 #     Modjtaba Shokrian Zini, Alain Delgado, Roberto dos Reis, Pablo A. M. Casares,
 #     Jonathan E. Mueller, Arne-Christian Voigt, Juan Miguel Arrazola,
 #     "Quantum simulation of battery materials using ionic pseudopotentials".
-#     `	arXiv:2302.07981 (2023)
-#     <https://arxiv.org/abs/2302.07981>`__
+#     `arXiv:2302.07981 (2023) <https://arxiv.org/abs/2302.07981>`__
```
Comment (obliviateandsurrender) [inline demonstrations/tutorial_resource_estimation.py:305]: [optional] Would we want to change this link to that of the published version? you may ignore it if not. 

```suggestion
#     `Quantum 7, 1049 (2023) <https://doi.org/10.22331/q-2023-07-10-1049>`__
```

**Review by obliviateandsurrender — APPROVED** [review]
Thanks @soranjh! Just a minor comment, otherwise looks good!

File: demonstrations/tutorial_resource_estimation.py [hunk 2]
```diff
@@ -302,8 +302,7 @@
 #     Modjtaba Shokrian Zini, Alain Delgado, Roberto dos Reis, Pablo A. M. Casares,
 #     Jonathan E. Mueller, Arne-Christian Voigt, Juan Miguel Arrazola,
 #     "Quantum simulation of battery materials using ionic pseudopotentials".
-#     `	arXiv:2302.07981 (2023)
-#     <https://arxiv.org/abs/2302.07981>`__
+#     `Quantum 7, 1049 (2023) <https://doi.org/10.22331/q-2023-07-10-1049>`__
```
Comment (soranjh) [inline demonstrations/tutorial_resource_estimation.py:305]:
```suggestion
#     `Quantum 7, 1049 (2023) <https://quantum-journal.org/papers/q-2023-07-10-1049/>`__

```

**Review by ddhawan11 — APPROVED** [review]

**Pull request merged**